.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "helium-java",
+
+  "settings": { "terminal.integrated.shell.linux": "/bin/bash" },
+
+	// Open port 4120 by default
+  "forwardPorts": [4120],
+
+  // Install extensions
```

Comment:
Should we get from Joseph the correct Linter to add as well? I think it was Sonar

Reply:
There's an open task to fix the F5 process, so we can address as part of that